packages/ui-extensions/src/shared.ts

```diff
@@ -37,7 +37,7 @@ export type ApiVersion =
  *
  * * [`collect_buyer_consent.customer_privacy`](https://shopify.dev/docs/api/checkout-ui-extensions/configuration#collect-buyer-consent): the extension can register buyer consent decisions that will be honored on Shopify-managed services.
  *
- * * `iframe.sources`: the extension can embed an external URL in an iframe.
+ * * [`iframe.sources`](https://shopify.dev/docs/api/checkout-ui-extensions/configuration#iframe): the extension can embed an external URL in an iframe.
```

Comment:
- [ ]  @rcaplanshopify  made these changes
 - [ ]   @rcaplanshopify  approves these changes for the 2024-10 stable release.
[adds iframe capabilities to types](https://github.com/Shopify/checkout-web/commit/93a5915dcef7abbef622833e12f8e6c0948c4fe6)